test/multipart_parser/parser_test.rb

```diff
@@ -1,6 +1,6 @@
 require 'test/unit'
-require "multipart_parser/parser"
-require "fixtures/multipart"
+require File.dirname(__FILE__) + "/../../lib/multipart_parser/parser"
```

Comment:
Could you explain this change?

I have not done much ruby in recent years, so I do not know whether there have been any changes as to how to import modules.

Reply:
Hi
I have no idea with it.
When I run the test, it failed by no such file error for requiring it.
However, reader_test.rb was able to pass the test. Therefore I ported the code from reader_test.rb to parser_test.rb.

Regards